app/build.gradle

```diff
@@ -1,19 +1,22 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
+    id 'androidx.navigation.safeargs.kotlin'
```

Comment:
OMG! too much files for one code review
Try to split it next time into several request. If it would be a real case, it would take 3 hours to check it all